controllers/article-controller.js

```diff
@@ -8,7 +8,7 @@ exports.getAllArticles = async (req, res) => {
     const foundArticles = await Article.findMany()
     return res.json({ articles: foundArticles })
   } catch (err) {
-    console.log(err)
+    return
```

Comment:
This silently swallows the error which will make debugging really difficult. As a general rule don't do this, pass the error to some error handlers that can identify the error and log it if needs be.